ui/src/css/pages/_View.scss

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * TODO(gimite): Consider sharing some rules with _Results.scss because they
+ * are similar.
+ */
+
+.view-body {
+  /* The 100px bottom margin ensures that the page can be scrolled far enough to
+   * show the footer text without it being covered by the new record FAB. */
+  margin: 16px 0 100px;
+  text-align: center;
+}
+
+.view-card {
```

Comment:
Any reason not to use the Material React library's [Card component](https://github.com/material-components/material-components-web-react/tree/master/packages/card) for this? We're using it on the global homepage (for repo link cards and the third-party devs/responders cards).

Reply:
I basically copied implementation of _Results.scss because this card looks similar to the search result card (though I made some modification). So can I ask you back: Why didn't you use Card component in Results.js?

Reply:
Ha, fair question.
IIRC, the reason I didn't use a Card for Results.js is because the results page doesn't have a separate card for each result, it just has a card-shaped wrapper around everything with horizontal rules between them. It didn't seem right to use a Card for a list of items, and [the MDC web docs](https://github.com/material-components/material-components-web/tree/master/packages/mdc-card) say that "Cards contain content and actions about a single subject." That means the wrapper around results shouldn't be a card because it's about many subjects, but we could use it for an individual result if we wanted.
How much of the _Results.scss CSS did you copy, and what did you need to change? If it'd be practical for them to share some of their CSS, that seems like a good reason to do it that way instead of using Card component, but if they're not sharing classes then IMO it should probably use a Card.

Reply:
I needed these changes:
* Support of sections without a photo:
![thgvqfgsrhs](https://user-images.githubusercontent.com/15363/52827092-bb182f80-3106-11e9-9f95-a42419205b19.png)
* Rename of classes (since it's not a list)
* (A change in HTML not CSS but) Use div instead of ul/li (because ul/li is weird here)

So I believe I can rewrite results page with CSS classes defined here (but not vice versa). Does that sound good? If so, can I submit this PR for now and make that change later?

Reply:
Yeah sure.